src/lib/jsonParse/jsonParse.cpp

```diff
@@ -557,7 +556,7 @@ std::string jsonParse
     if (res != "OK")
     {
       std::string details = std::string("JSON parse error: '") + res + "'";
-      alarmMgr.badInput(clientIp, details);
+      alarmMgr.badInput(clientIp, details, "");
```

Comment:
Makes sense split the details? (similar as described above)

Reply:
Fixed in f1d444ca5f9c1e526bce85c141febb468e447eb9